cmd/collectors/ems/ems.go

```diff
@@ -87,6 +88,8 @@ func (e *Ems) Init(a *collector.AbstractCollector) error {
 	e.InitProp()
 	// init ems props
 	e.InitEmsProp()
+	// init bookend resolve ems map. This is reverse map, [Resolving ems]:[Issuing ems]
```

Comment:
nice comment, would be better to move this to the struct definition above instead of where it is instantiated